src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -16,6 +16,7 @@
 
 using System.Diagnostics.Metrics;
 using System.Reflection;
+using Diagnostics = System.Diagnostics;
```

Comment:
Do we need an alias for the namespace?

Reply:
I've created a static var for the current process and removed the alias.